.github/dependabot.yml

```diff
@@ -26,49 +26,13 @@ updates:
     open-pull-requests-limit: 20
 
   - package-ecosystem: nuget
-    directory: /examples/AspNetCoreMvc
+    directory: /examples/cli
     schedule:
       interval: "daily"
     open-pull-requests-limit: 20
 
   - package-ecosystem: nuget
-    directory: /examples/BindingRedirect
-    schedule:
-      interval: "daily"
-    open-pull-requests-limit: 20
-
-  - package-ecosystem: nuget
-    directory: /examples/ConsoleApp
-    schedule:
-      interval: "daily"
-    open-pull-requests-limit: 20
-
-  - package-ecosystem: nuget
-    directory: /examples/ConsoleApp.SelfBootstrap
-    schedule:
-      interval: "daily"
-    open-pull-requests-limit: 20
-
-  - package-ecosystem: nuget
-    directory: /examples/CoreAppOldReference
-    schedule:
-      interval: "daily"
-    open-pull-requests-limit: 20
-
-  - package-ecosystem: nuget
-    directory: /examples/OldReference
-    schedule:
-      interval: "daily"
-    open-pull-requests-limit: 20
-
-  - package-ecosystem: nuget
-    directory: /examples/OpenTracingLibrary
-    schedule:
-      interval: "daily"
-    open-pull-requests-limit: 20
-
-  - package-ecosystem: nuget
-    directory: /examples/Vendor.Distro
+    directory: /examples/srv
```

Comment:
We are in the .NET world, no Go-lang. ;)
No abbreviations needed,

Reply:
Addressed https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/1708/commits/cc20430edf4e6863964f62ca94727b6891599792